recipe/bld.bat

```diff
@@ -74,24 +72,8 @@ if errorlevel 1 exit /b 1
 :: ctest -C Release
 :: if errorlevel 1 exit 1
 
-:: Windows 32-bit seems to want to run the wrong zlib DLL
-:: We have 2 zlib-1.2.11 build #3 for win-64:
-:: zlib-1.2.11-h3cc03e0_3.tar.bz2	120 KB	2018-11-21 15:50:01 +0000	928fa9aa513a1d08016a4079e2c52706a3fd29a5f2acd16eabec5e8ab9599478	774d60f873e10acc14be0d239c18d614
-:: zlib-1.2.11-h62dcd97_3.tar.bz2	128 KB	2018-11-21 15:46:22 +0000	3a5148e6b79ae0ec581df96d97d248d38366db49109f1726eda76639551745be	6f96fd91475cc78aabf76d2e1a9ed91f
-:: https://repo.anaconda.com/pkgs/main/win-64/zlib-1.2.11-h3cc03e0_3.tar.bz2
-:: https://repo.anaconda.com/pkgs/main/win-64/zlib-1.2.11-h62dcd97_3.tar.bz2
-
 :: Test extracting a 7z. This failed due to not using the multi-threaded DLL runtime, fixed by 0009-CMake-Force-Multi-threaded-DLL-runtime.patch
-:: %BUILD_PREFIX%\Library\bin\curl.exe -SLO http://download.qt.io/development_releases/prebuilt/llvmpipe/windows/opengl32sw-64-mesa_12_0_rc2.7z
-:: powershell -command "& { (New-Object Net.WebClient).DownloadFile('http://download.qt.io/development_releases/prebuilt/llvmpipe/windows/opengl32sw-64-mesa_12_0_rc2.7z', 'opengl32sw-64-mesa_12_0_rc2.7z') }"
-:: if errorlevel 1 exit 1
-:: %LIBRARY_BIN%\bsdtar.exe -xf opengl32sw-64-mesa_12_0_rc2.7z
-:: if errorlevel 1 exit 1
-
-echo Trying to run %LIBRARY_BIN%\bsdcat.exe --version
-%LIBRARY_BIN%\bsdcat.exe --version
+powershell -command "& { (New-Object Net.WebClient).DownloadFile('http://download.qt.io/development_releases/prebuilt/llvmpipe/windows/opengl32sw-64-mesa_12_0_rc2.7z', 'opengl32sw-64-mesa_12_0_rc2.7z') }"
```

Comment:
I don't think we should include pre-compiled binaries from a third party in our package like this. At the very least, it would have to be done via HTTPS or verified with a cryptographic hash. It was commented out above, do we need this and what for?

Reply:
Oh, never mind me, this is libarchive. It's a funny way to test it, TBH.

Reply:
I'm not sure this should happen in the build script, though. What if we inadvertently add test fixtures to the package?

Reply:
You are right. We already have test commands for that purpose.